JASP-Tests/R/tests/testthat/test-summarystatscorrelationbayesianpairs.R

```diff
@@ -1,29 +1,16 @@
-context("SummaryStatsCorrelationBayesianPairs")
+context("Summary Statistics Bayesian Correlation Pairs")
```

Comment:
Can you undo the changes you made to this file?
This is what fails the unit tests right now and considering you didn't rewrite correlation these changes seem unnecessary.

Reply:
corrected.